### PR TITLE
fix(user menu): full height user menu

### PR DESCRIPTION
**Pull request by manojava-gk**
## Description

display full width menu. move menu footer to the bottom section.
no custom style for sub navigation header

## Why

new change request

## Issue

NA

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [ ] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [ ] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [ ] I have added tests that prove my changes work
- [ ] I have checked that new and existing tests pass locally with my changes
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/shared/MobileMenu/index.tsx [hunk 1]
```diff
@@ -89,45 +89,57 @@ export const MobileMenu = ({
...
+                  Menu={MobileMenu}
+                  onSelect={onSelectItem}
+                  onClick={onClick}
+                  key={uniqueId('Menu')}
+                />
+              ))}
+            </List>
+            {divider && <Divider sx={{ margin: spacing(0, 1) }} />}
+            <ProfileLink to={'/account'} />
+            <NotificationLink to={'/notifications'} />
```
Comment (lavanya-bmw) [inline src/components/shared/MobileMenu/index.tsx:119]: Please remove hardcoded link and add this from constants

**Review by lavanya-bmw — CHANGES_REQUESTED** [review]
Please add changelog

**Review by lavanya-bmw — APPROVED** [review]
lgtm

**Comment by manojava-gk** [comment]
@evegufy can you merge this PR?

**Comment by evegufy** [comment]
@manojava-gk is there no issue for this change?

**Comment by manojava-gk** [comment]
> @manojava-gk is there no issue for this change?

No. I have tested this in the local

**Comment by manojava-gk** [comment]
@oyo @nidhigarg-bmw @lavanya-bmw Kindly review once again

**Review by nidhigarg-bmw — APPROVED** [review]
lftm

**Review by oyo — APPROVED** [review]

**Pull request merged**